src/package_io/parse_input.star

```diff
@@ -255,16 +255,15 @@ def enrich_mev_extra_params(parsed_arguments_dict, mev_prefix, mev_port):
 
 	mev_participant = {
 		"el_client_type": "geth",
-		# TODO replace with actual when flashbots/builder is published
-		"el_client_image": "h4ck3rk3y/builder",
+		"el_client_image": "flashbots/builder:sha-ffeb673",
 		"el_client_log_level":    "",
 		"cl_client_type":         "lighthouse",
 		# THIS overrides the beacon image
 		"cl_client_image":        "sigp/lighthouse",
 		"cl_client_log_level":    "",
 		"beacon_extra_params":    ["--builder={0}".format(mev_url), "--always-prepare-payload", "--prepare-payload-lookahead", "12000"],
 		# TODO(maybe) make parts of this more passable like the mev-relay-endpoint & forks
-		"el_extra_params": ["--builder",  "--builder.remote_relay_endpoint=http://mev-relay-api:9062", "--builder.beacon_endpoints=http://cl-{0}-lighthouse-geth:4000".format(num_participants+1), "--builder.bellatrix_fork_version=0x30000038", "--builder.genesis_fork_version=0x10000038", "--builder.genesis_validators_root={0}".format(package_io.GENESIS_VALIDATORS_ROOT_PLACEHOLDER),  "--miner.extradata=\"Illuminate Dmocratize Dstribute\"", "--miner.algotype=greedy"] + parsed_arguments_dict["mev_params"]["mev_builder_extra_args"],
+		"el_extra_params": ["--builder",  "--builder.remote_relay_endpoint=http://mev-relay-api:9062", "--builder.beacon_endpoints=http://cl-{0}-lighthouse-geth:4000".format(num_participants+1), "--builder.bellatrix_fork_version=0x30000038", "--builder.genesis_fork_version=0x10000038", "--builder.genesis_validators_root={0}".format(package_io.GENESIS_VALIDATORS_ROOT_PLACEHOLDER),  "--miner.extradata=\"Illuminate Dmocratize Dstribute\"", "--builder.algotype=greedy"] + parsed_arguments_dict["mev_params"]["mev_builder_extra_args"],
```

Comment:
its pretty hard to read individual params, could you make this a multi line instead?